packages/core/src/domain/configuration/tags.ts

```diff
@@ -32,7 +32,9 @@ export function buildTag(key: string, rawValue: string) {
   const valueSizeLimit = TAG_SIZE_LIMIT - key.length - 1
 
   if (rawValue.length > valueSizeLimit || FORBIDDEN_CHARACTERS.test(rawValue)) {
-    display.warn(`${key} value doesn't meet tag requirements and will be sanitized`)
+    display.warn(
+      `${key} value doesn't meet tag requirements and will be sanitized. More details: https://docs.datadoghq.com/getting_started/tagging/#defining-tags`
```

Comment:
**💭 thought:** ‏We now have 5 doc URLs showing up in the RUM bundle. Maybe a constant could save a few bytes:

```ts
const DOCS_ORIGIN = 'https://docs.datadoghq.com'

...
`${DOCS_ORIGIN}/getting_started/tagging/#defining-tags`
```